internal/gengapic/genrest.go

```diff
@@ -529,24 +529,85 @@ func (g *generator) pagingRESTCall(servName string, m *descriptor.MethodDescript
 	p := g.printf
 
 	inType := g.descInfo.Type[m.GetInputType()].(*descriptor.DescriptorProto)
-	// outType := g.descInfo.Type[m.GetOutputType()].(*descriptor.DescriptorProto)
+	outType := g.descInfo.Type[m.GetOutputType()].(*descriptor.DescriptorProto)
 
 	inSpec, err := g.descInfo.ImportSpec(inType)
 	if err != nil {
 		return err
 	}
 
-	// outSpec, err := g.descInfo.ImportSpec(outType)
-	// if err != nil {
-	// 	return err
-	// }
+	outSpec, err := g.descInfo.ImportSpec(outType)
+	if err != nil {
+		return err
+	}
+	info, err := getHTTPInfo(m)
+	if err != nil {
+		return err
+	}
+	if info == nil {
+		return errors.E(nil, "method has no http info: %s", m.GetName())
+	}
+	max := "math.MaxInt32"
+	ps := "int32(pageSize)"
+
+	verb := strings.ToUpper(info.verb)
+
 	pageSizeFieldName := snakeToCamel(pageSize.GetName())
+	g.imports[pbinfo.ImportSpec{Path: "google.golang.org/protobuf/encoding/protojson"}] = true
```

Comment:
Nit: move this import to the bottom of this function with the other utility imports